### PR TITLE
Tpm2 policycphash

### PR DESCRIPTION
**Pull request by idesai**
Progresses #859

### CODE REVIEW

File: lib/tpm2.c [hunk 1]
```diff
@@ -2079,3 +2142,80 @@ tool_rc tpm2_gettime(ESYS_CONTEXT *ectx,
...
+    }
+
+    //CpBuffer
+    memcpy(to_hash + offset, command_parameters, command_parameters_size);
+
+    //cpHash
+    tool_rc rc = tool_rc_success;
+    bool result = tpm2_openssl_hash_compute_data(halg, to_hash, to_hash_len,
+        cp_hash);
+    free(to_hash);
```
Comment (idesai) [inline lib/tpm2.c]: Should be Esys_Free

**Comment by williamcroberts** [comment]
@idesai can you rebase this?

File: .ci/download-deps.sh [hunk 2]
```diff
@@ -3,8 +3,8 @@
 
 function get_deps() {
 
-	TSS_VERSION="2.3.1"
-	ABRMD_VERSION="2.3.0"
+	TSS_VERSION=master
+	ABRMD_VERSION=master
```
Comment (williamcroberts) [inline .ci/download-deps.sh]: Do we need these set to master or can we bump them to a stable release for the feature you need?

Comment (idesai) [reply .ci/download-deps.sh]: @williamcroberts the change for retrieving the sapi handle is in tpm2-tss master. I also see it is in v2.4rc1. Perhaps I can tag this as 2.4 and wait until the rc becomes a full release.

**Pull request merged**